### PR TITLE
DF-520: Change legal terms link on the registration page

### PR DESCRIPTION
**Pull request by birom**

### CODE REVIEW

File: alv-portal-ui/src/app/shared/legal-terms/legal-terms.service.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    case 'en':
+      return legalTerms.linkEn;
+    case 'de':
+      return legalTerms.linkDe;
+    case 'fr':
+      return legalTerms.linkFr;
+    case 'it':
+      return legalTerms.linkIt;
+    default:
+      throw Error('Failed to load legal terms url!');
```
Comment (pavax) [inline alv-portal-ui/src/app/shared/legal-terms/legal-terms.service.ts]: if we throw an Error here then the message is kina misleading. should be something like: 'can not extract legalTerm Url for language: currentLanguage)

File: alv-portal-ui/src/app/shared/legal-terms/legal-terms.service.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+@Injectable({
+  providedIn: 'root'
+})
+export class LegalTermsService {
+
+  constructor(private i18nService: I18nService,
+              private legalTermsManagementRepository: LegalTermsManagementRepository) {
+  }
+
+  getCurrentLegalTermsUrl(): Observable<string> {
```
Comment (pavax) [inline alv-portal-ui/src/app/shared/legal-terms/legal-terms.service.ts]: why is it named currentLegalTerms? and not just legalTerms

File: alv-portal-ui/src/app/shared/legal-terms/legal-terms.service.ts (hunk 1 above)
Comment (birom) [reply alv-portal-ui/src/app/shared/legal-terms/legal-terms.service.ts]: German is returned as default.

File: alv-portal-ui/src/app/shared/legal-terms/legal-terms.service.ts (hunk 2 above)
Comment (birom) [reply alv-portal-ui/src/app/shared/legal-terms/legal-terms.service.ts]: thx, fixed

**Pull request merged**